javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj) {
+
+  let unorderedList = document.createElement('ol')
+
+  for(const [key, value] of Object.entries(obj)){
+    let list = document.createElement("li")
+    list.textContent = `${key}: ${value}`
+    unorderedList.appendChild(list);
+  }  
+};
```

Comment:
Hi there! 

Nice work so far on these :) Regarding question for this: Take a look at challenge instructions one last time and the tests. What do the tests expect to see here? 

You need to return an array of key value pairs. You just need to do a bit of tweaking here. Grab a classmate or TA to get ya going 👍